selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOfSubOrganizationViewsTest.java

```diff
@@ -113,40 +106,36 @@ public void testOrganizationListComponents() {
     assertTrue(organizationListPage.getValues(NAME).contains(childOrganization.getQualifiedName()));
   }
 
-  @Test(priority = 2)
-  public void testParentOrganization() {
-    organizationListPage.clickOnOrganization(parentOrganization.getName());
+  @Test
+  public void parentOrganizationViewsTest() {
```

Comment:
Please, see my notices [here](https://github.com/eclipse/che/pull/7312/files#r150507000) and [here](https://github.com/eclipse/che/pull/7312/files#r150517981).